Console/Command/RetryYotpoSync.php

```diff
@@ -127,6 +127,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $this->resyncAllEntities($output);
         } else {
             $yotpoEntity = $input->getOption(self::YOTPO_ENTITY);
+            if (is_array($yotpoEntity)) {
+                $yotpoEntity = implode('', $yotpoEntity);
+            }
```

Comment:
What is this for?

Reply:
Hi Eran,

If we don't implode it , we will get the below code validation error from PHPSTAN tool                     
                                 
At line 146  
Binary operation "." between 'Entity ' and array<string>|bool|string|null results in an error.

This is because Symfony\Component\Console\Input::getOption($name) method's signature is @return string|string[]|bool|null The option value

We had put the implode action to get rid of this validation error.

I have now changed the implementation in a different way to consider the case of an array and also to resolve the above mentioned error. Please check the latest commit

Reply:
Hi Manaf, we should not add meaningless logic for linter validations to pass, rather code in a way that expresses the BL only.
Would love to see your new implementation.
Can you please share the updated PR (on top of this)?

Reply:
This is implemented at https://github.com/YotpoLtd/magento2-module-core/pull/136